service/src/main/resources/db/data/hameenkyro/afterMigrate__service_need_options.sql

```diff
@@ -17,6 +17,7 @@ VALUES
     ('f5d32585-2c78-4434-95d6-30b446db7d4d', 'Palveluseteli Varhaiskasvatus yli 140 tuntia / kuukausi', 'Palveluseteli Varhaiskasvatus yli 140 tuntia / kuukausi', 'Voucher value Daycare - over 140 h per month', 'DAYCARE', FALSE, 1.0, 1.0, 1.0, 1.0, 1.0, 40, NULL, 210, FALSE, FALSE, 'Varhaiskasvatus yli 140 tuntia / kuukausi', 'Varhaiskasvatus yli 140 tuntia / kuukausi', 'Varhaiskasvatus yli 140 tuntia / kuukausi', 'Varhaiskasvatus yli 140 tuntia / kuukausi', TRUE, TRUE, 6),
     ('e1063bee-c19d-469d-85a5-6b0350872d76', 'Tilapäinen varhaiskasvatus', 'Tilapäinen varhaiskasvatus', 'Temporary daycare', 'TEMPORARY_DAYCARE', TRUE, 1.0, 1.0, 1.75, 1.0, 1.75, 40, NULL, NULL, FALSE, FALSE, 'Tilapäinen varhaiskasvatus', 'Tilapäinen varhaiskasvatus', 'Tilapäinen varhaiskasvatus', 'Tilapäinen varhaiskasvatus', TRUE, TRUE, NULL),
     ('94e44ef1-106b-401d-81b6-8e5c31cd0437', 'Esiopetus', 'Esiopetus', 'Preschool', 'PRESCHOOL', TRUE, 0.0, 0.5, 0.5, 0.5, 0.5, 0, NULL, NULL, TRUE, TRUE, 'Esiopetus', 'Esiopetus', 'Esiopetus', 'Esiopetus', TRUE, TRUE, NULL),
+    ('1a4b0997-b83d-44ec-8bd9-98b12e5d6d04', 'Valmistava opetus', 'Valmistava opetus', 'Preparatory education', 'PREPARATORY', TRUE, 0.0, 0.5, 0.5, 0.5, 0.5, 0, NULL, NULL, TRUE, TRUE, 'Valmistava opetus', 'Valmistava opetus', 'Valmistava opetus', 'Valmistava opetus', TRUE, TRUE, NULL),
```

Comment:
This is still missing options for `PREPARATORY_DAYCARE` & `PREPARATORY_DAYCARE_ONLY`, good starting point is to copy from `PRESCHOOL_DAYCARE` & `PRESCHOOL_DAYCARE_ONLY`.

Reply:
Sorry, I still get these terms mixed up... 

Added
* PREPARATORY_DAYCARE -  'Valmistava opetus ja liittyvä varhaiskasvatus', 'Preparatory education'
* PREPARATORY_DAYCARE_ONLY - 'Valmistavan opetuksen liittyvä varhaiskasvatus', 'Preparatory education'

(Couldn't find any other English translation besides 'Preparatory education')